$NetBSD: patch-mkworddic_compound.t,v 1.1 2026/09/15 13:00:15 wiz Exp $

Fix wrong kanji, wrong readings and malformed part-of-speech tags.

The conversions corrected here give a different word from the one the reading
asks for:

	уБТуВУуБЛуВУуБЧуВЗуБЖуВБуБД	чОДщЦвши╝цШО	-> чОДщЦвчЕзцШО
	уБУуБЖуБЛуБДуБпуВУуБжуБД	щлШщЪОхИдхоЪ	-> хЕмщЦЛхИдхоЪ
	уБЧуВЗуБЖуБЛуВИуБЖуБЩуБД	ц╢ИхМЦчФиц░┤	-> ц╢ИчБлчФиц░┤
	уБ▓уБкуВУуБШуВГуБПуБЧуВГ	щЭЮщЫгх╝▒шАЕ	-> щБ┐щЫгх╝▒шАЕ
	уБ▓уБкуВУуБЯуБДуВН		щЭЮщЫгщААш╖п	-> щБ┐щЫгщААш╖п
	уВЖуБ╛уБПуБДуБШуВЗуБЖ		ц▓╣шЖЬф╗еф╕К	-> ц▓╣шЖЬчХ░х╕╕
	уБЬуВУуБНуВЗуБПуВНуБПуБКуВУ	цнйщБУцХ┤хВЩ	-> хЕицЫ▓щМ▓щЯ│

The last one had been given the entry for a different word entirely; цнйщБУцХ┤хВЩ
is added back under its own reading, уБ╗уБйуБЖуБЫуБДуБ│.

Two readings were wrong: уБВуБДуБЪуБЧуБгуБН for ф╝Ъц┤ец╝ЖхЩи, which is уБВуБДуБе, and
уБЛуВУуБЩуБДуБНуВУуБЮуБП for щЦвц░┤щЗСх▒Ю, which is уБЫуБНуБЩуБД.  Seven entries carried a
malformed part-of-speech tag (no leading #, or #35, #T5, #T#5 where #T30 or
#T35 was meant); mkworddic takes the tag only from a token that starts with #
and names a known type, so those compounds were never registered.

Fedora has carried these since 2010, and anthy-unicode -- the fork that is
still maintained -- has every one of them.

--- mkworddic/compound.t.orig
+++ mkworddic/compound.t
@@ -8,7 +8,6 @@
 двддд╖дуд╗ддд╖дє #T35 #_4░ж╝╥_4└║┐└
 двддд╖дчджд╖дєд└дє #T35 #_5┴ъ└н_4┐╟├╟
 двддд╖дчдждсдд #T35 #_5░ж╛╬_2╠╛
-двддд║д╖д├дн #T35 #_3▓ё─┼_3╝┐┤я
 двддд╝дєд╖дхд┐д├д╚ #CN еведе╝еєе╖ехе┐е├е╚
 двддд╛джд▓дн #T35 #_4░ж┴■_2╖р
 двддд┴ддд└дд #T35 #_3░ж├╬_3░х┬ч
@@ -20,6 +19,7 @@
 двддд┴д│джд╕дчдж #T35 #_3░ж├╬_5╣й╛ь
 двддд┴д│джд└дд #T35 #_3░ж├╬_4╣й┬ч
 двддд┴д╖дхд├д╖дє #T35 #_3░ж├╬_5╜╨┐╚
+двддд┼д╖д├дн #T35 #_3▓ё─┼_3╝┐┤я
 двддд┼д└дддмдп #T35 #_3▓ё─┼_4┬ч│╪
 двддд┼д╞д─д╔дж #T35 #_3▓ё─┼_4┼┤╞╗
 двддд┼д╞д─д╔джд╗дє #T35 #_3▓ё─┼_6┼┤╞╗└■
@@ -1987,7 +1987,7 @@
 ддд├д╤дєдшдфдпдлддд╖ #T35 #_4░ь╚╠_3═╜╠є_3│л╗╧
 ддд├д╤дєдъдчдп #T35 #_4░ь╚╠_3╬╧
 ддд├д╤дєдъдчд│дж #T35 #_4░ь╚╠_4╬╣╣╘
-ддд├д╤дєдъдшдж #35 #_4░ь╚╠_3═°═╤
+ддд├д╤дєдъдшдж #T30 #_4░ь╚╠_3═°═╤
 ддд├д╤дєдъдшджд╖ду #T35 #_4░ь╚╠_3═°═╤_2╝╘
 ддд├д╤дєдъдшджд╖дудрдн #T35 #_4░ь╚╠_3═°═╤_2╝╘_2╕■
 ддд├д╤дєдъдэдє #T35 #_4░ь╚╠_3═¤╧└
@@ -9714,7 +9714,6 @@
 длдєд╕дєд║дтдж #T35 #_4┤л┐╩_3┴ъ╦╨
 длдєд╕дєд┴дчдж #T35 #_4┤л┐╩_3─в
 длдєд╣дддлд╖дч #T35 #_4┤з┐х_3▓╒╜ъ
-длдєд╣дддндєд╛дп #T35 #_4┤╪┐х_4╢т┬░
 длдєд╣дддодч #T35 #_4┤и┐х_2╡√
 длдєд╣ддд╨д╖дч #T35 #_4┤з┐х_3╛ь╜ъ
 длдєд╣дждлддд╗дн #T35 #_4┤╪┐Ї_4▓Є└╧
@@ -10235,7 +10234,7 @@
 длдєд╓д╕дєд╕ #T35 #_3┤┤╔Ї_3┐═╗Ў
 длдєд╓д─дф #T35 #_4┤е╩к_1▓░
 длдєд╓дъ #T35 #_2┤и_2е╓еъ
-длдєд╓дъдхджд╖дхд─ #T#5 #_3┤┤╔Ї_6╬о╜╨
+длдєд╓дъдхджд╖дхд─ #T30 #_3┤┤╔Ї_6╬о╜╨
 длдєд╓дєдпдєд╔дпд┐дд #T35 #_4┤┴╩╕_4╖▒╞╔_2┬╬
 длдєд╓дєд╞дн #T35 #_4┤┴╩╕_2┼к
 длдєд╫джд▓дн #T35 #_4┤░╔ї_2╖р
@@ -14320,7 +14319,7 @@
 дндчджд╚д╖дєд╓дє #KK #_4╡■┼╘_4┐╖╩╣
 дндчджд╚д╖дєд╓дєд╧дд #T35 #_4╡■┼╘_4┐╖╩╣_2╟╒
 дндчджд╚д╕дхдж #T35 #_4╡■┼╘_3├ц
-дндчджд╚д╕дхдждлдєд╕д╔джд╖дуд╔дж T35 #_4╡■┼╘_5╜─┤╙_5╝л╞░╝╓_2╞╗
+дндчджд╚д╕дхдждлдєд╕д╔джд╖дуд╔дж #T35 #_4╡■┼╘_5╜─┤╙_5╝л╞░╝╓_2╞╗
 дндчджд╚д╕дє #T35 #_4╡■┼╘_2┐═
 дндчджд╚д╗дщд▀д├дп #KK #_4╡■┼╘_5е╗еще▀е├еп
 дндчджд╚д┐ддд╖дчджд╞дє #T35 #_4╡■┼╘_5┬ч╛▐_2┼╡
@@ -19234,7 +19233,7 @@
 д▓дєдлдєд░д┴ #T35 #_4╕╝┤╪_2╕¤
 д▓дєдлдєд▓дєд╛дж #T35 #_2╕║_2┤╢_4╕╜┴№
 д▓дєдлдєд╡дн #T35 #_4╕╝┤╪_2└ш
-д▓дєдлдєд╖дчдждсдд #T35 #_4╕╝┤╪_5╛┌╠└
+д▓дєдлдєд╖дчдждсдд #T35 #_4╕╝┤╪_5╛╚╠└
 д▓дєдлдєд╣дж #T35 #_2╕╢_4┤╪┐Ї
 д▓дєдлдєд▐ди #T35 #_4╕╝┤╪_2┴░
 д▓дєдлдєдъд╦дє #T35 #_2╕╜_3┤╔═¤_2┐═
@@ -19964,7 +19963,7 @@
 д▓дєдъдчджд┴дчджд┐д─ #T35 #_5╕╢╬┴_5─┤├г
 д▓дєдэдждддє #T35 #_4╕╡╧╖_2▒б
 д▓дєдэдпд║д╖ #T35 #_4╕╡╧╜_2╝ў╗╩
-д▓дєдэд╗дє #T5 #_2╕╜_3╧й└■
+д▓дєдэд╗дє #T35 #_2╕╜_3╧й└■
 д▓дєдэдєдпдждлдє #T35 #_4╕└╧└_4╢ї┤╓
 д▓дєдэдєд╕дцдж #T35 #_4╕└╧└_3╝л═│
 д▓дєдэдєд╗ддд▓дє #T35 #_4╕└╧└_4└й╕┬
@@ -20163,7 +20162,7 @@
 д│дждлддд╚дждэдє #T35 #_4╕°│л_4╞д╧└
 д│дждлддд╩д▐д█джд╜дж #T35 #_4╕°│л_2└╕_4╩№┴ў
 д│дждлддд╧дє #T35 #_4╣╥│д_2╚╔
-д│дждлддд╧дєд╞дд #T35 #_4╣т│м_4╚╜─ъ
+д│дждлддд╧дєд╞дд #T35 #_4╕°│л_4╚╜─ъ
 д│дждлддд╨дє #T35 #_4╕°│л_2╚╟
 д│дждлддд╙ #T35 #_4╕°│л_1╞№
 д│дждлддд█джд╜дж #T35 #_4╕°│л_4╩№┴ў
@@ -21650,7 +21649,7 @@
 д│джд╞ддд╧ #T35 #_4╣╬─ъ_1╟╔
 д│джд╞дндлддд┤д█д▒дє #T35 #_4╕°┼к_3▓Ё╕ю_3╩▌╕▒
 д│джд╞дндлдєдндє #T35 #_4╕°┼к_4┤╣╢т
-д│джд╞дндлдєдш T35 #_4╕°┼к_3┤╪═┐
+д│джд╞дндлдєдш #T30 #_4╕°┼к_3┤╪═┐
 д│джд╞дндндлдє #T35 #_4╕°┼к_3╡б┤╪
 д│джд╞днднд╗дд #T35 #_4╕°┼к_3╡м└й
 д│джд╞дндндєдцдж #T35 #_4╕°┼к_4╢т═╗
@@ -30752,7 +30751,6 @@
 д╖дхдєд╕дхджд╗дєд┤дпд╕д└дд #T35 #_6╜╒╜й_4└я╣ё_3╗■┬х
 д╖дхдєд╕дхджд╬д▐ #T35 #_7╜╒╜йд╬_1┤╓
 д╖дхдєд╞ддд┐ддд╡дп #T35 #_5╜╓─ф_4┬╨║Ў
-д╖дхдєд╚джд░дє #T35 #_5╜┘┼ь_2╖┤
 д╖дхдєд╤д─дъдчдп #T35 #_5╜╓╚п_3╬╧
 д╖дхдєд╤д─дъдчдпд╖дчджд╓ #T35 #_5╜╓╚п_3╬╧_4╛б╔щ
 д╖дхдєд╙дєд╗дд #T35 #_5╜╙╔╥_2└н
@@ -30839,7 +30837,7 @@
 д╖дчдждлд╪дд #T35 #_3╛о_3▓╓╩┴
 д╖дчдждлд▄дж #T35 #_4╛╪▓┌_2╦╝
 д╖дчдждлдшдждддє #T35 #_4╛├▓╜_4═╫░ў
-д╖дчдждлдшджд╣дд #T35 #_4╛├▓╜_4═╤┐х
+д╖дчдждлдшджд╣дд #T35 #_4╛├▓╨_4═╤┐х
 д╖дчдждлдъдчдп #T35 #_4╛├▓╜_3╬╧
 д╖дчдждлдєдндлдє #T35 #_5╜■┤╘_3┤№┤╓
 д╖дчдждлдєднд▓дє #T35 #_5╜■┤╘_3┤№╕┬
@@ -40405,6 +40403,7 @@
 д╗днд╖дчдпд╚дж #T35 #_5└╓┐з_2┼Ї
 д╗днд╕д╥дчдж #T35 #_3└╩╝б_3╔╜
 д╗днд╕дхджд╕дн #T35 #_6└╓╜╜╗·_1┤·
+д╗днд╣дддндєд╛дп #KK #_4┤╪┐х_4╢т┬░
 д╗днд║ддд╖дчдж #T35 #_4└╘┐ё_3╛╔
 д╗днд║ддд╜дєд╖дчдж #T35 #_4└╘┐ё_5┬╗╜¤
 д╗днд║ддд╧дєд╖ду #T35 #_4└╘┐ё_4╚┐╝═
@@ -40670,7 +40669,7 @@
 д╗д├д┴дудпд╢дд #T35 #_5└▄├х_2║▐
 д╗д├д┴дудпд╢дд #T35 #_5└▄├х_2║р
 д╗д├д┴дудпд╓ #T35 #_5└▄├х_1╔Ї
-д╗д├д┴дудпд╓д╓дє T35 #_5└▄├х_3╔Ї╩м
+д╗д├д┴дудпд╓д╓дє #T35 #_5└▄├х_3╔Ї╩м
 д╗д├д┴дудпдсдє #T35 #_5└▄├х_2╠╠
 д╗д├д┴дудпдсдєд╗дн #T35 #_5└▄├х_4╠╠└╤
 д╗д├д┴дхдждвдє #T35 #_5└▐├я_2░╞
@@ -41942,7 +41941,7 @@
 д╝дєдндчдпд╣дж #T35 #_5┴┤╢╩_2┐Ї
 д╝дєдндчдпд┴дхдж #T35 #_5┴┤╢╩_3├ц
 д╝дєдндчдпд╨дє #T35 #_5┴┤╢╩_2╚╟
-д╝дєдндчдпдэдпдкдє #T35 #_5╩т╞╗_4└░╚ў
+д╝дєдндчдпдэдпдкдє #T30 #_5┴┤╢╩_4╧┐▓╗
 д╝дєдндєдвдєд╞ддд╗дд #T35 #_4┴▓╢с_6░┬─ъ└н
 д╝дєдндєд╗дє #T35 #_4┴▓╢с_2└■
 д╝дєдндєд╗дєд╕дчдж #T35 #_6┴▓╢с└■_3╛х
@@ -46498,7 +46497,7 @@
 д┴дхджд╨д┴д▒дє #KK #_5├ц╚н_2╕ж
 д┴дхджд╨дєддд│дж #T35 #_5├ц╚╫_3░╩╣▀
 д┴дхджд╨дєддд╕дчдж #T35 #_5├ц╚╜_4░╩╛х
-д┴дхджд╓д╕дхдждлдєд╕д╔джд╖дуд╔дж T35 #_4├ц╔Ї_5╜─┤╙_5╝л╞░╝╓_2╞╗
+д┴дхджд╓д╕дхдждлдєд╕д╔джд╖дуд╔дж #T35 #_4├ц╔Ї_5╜─┤╙_5╝л╞░╝╓_2╞╗
 д┴дхджд╓д╝дд #T35 #_4├ц╔Ї_2└к
 д┴дхджд╓д┴дп #T35 #_4├ц╔Ї_2├╧╢ш
 д┴дхджд╓д┴д█дж #T35 #_4├ц╔Ї_3├╧╩¤
@@ -57562,11 +57561,11 @@
 д╥д╩дєд┤дф #T35 #_3╚Є╞ё_2╛о▓░
 д╥д╩дєд╡дн #T35 #_3╚Є╞ё_2└ш
 д╥д╩дєд╖дидє #T35 #_3╚Є╞ё_3╗┘▒ч
-д╥д╩дєд╕дудпд╖ду #T35 #_3╚є╞ё_5╝х╝╘
+д╥д╩дєд╕дудпд╖ду #T35 #_3╚Є╞ё_5╝х╝╘
 д╥д╩дєд╕дхджд▀дє #T35 #_3╚Є╞ё_5╜╗╠▒
 д╥д╩дєд╕дч #T35 #_3╚Є╞ё_2╜ъ
 д╥д╩дєд╗дддлд─ #T35 #_3╚Є╞ё_4└╕│ш
-д╥д╩дєд┐дддэ #T35 #_3╚є╞ё_3┬р╧й
+д╥д╩дєд┐дддэ #T35 #_3╚Є╞ё_3┬р╧й
 д╥д╩дєд─дждэ #T35 #_3╚Є╞ё_3─╠╧й
 д╥д╩дєд╨д╖дч #T35 #_3╚Є╞ё_3╛ь╜ъ
 д╥д╩дєдцджд╔дж #T35 #_3╚Є╞ё_4═╢╞│
@@ -62585,6 +62584,7 @@
 д█д╚д▒д╡д▐ #T35 #_3╩й_2══
 д█д╔дждмдя #T35 #_3╩т╞╗_2┬ж
 д█д╔джд╕дчдж #T35 #_3╩т╞╗_3╛х
+д█д╔джд╗ддд╙ #T30 #_3╩т╞╗_3└░╚ў
 д█д╔дждьдн #T35 #_3╩ф╞│_2╬Є
 д█д╦дхджд╔джд╓д─ #T35 #_4╙о╞¤_4╞░╩к
 д█д╦дхджд╙дє #T35 #_4╙о╞¤_2╔╙
@@ -66264,7 +66264,6 @@
 дцд╜дждн #T35 #_3═в┴ў_1╡б
 дцд╜дждндлдє #T35 #_3═в┴ў_3╡б┤╪
 дцд╜джднд┴ #T35 #_3═в┴ў_2┤Ё├╧
-дцд╜джднд╔ #T35 #_3═в┴ў_2╡▒┼┘
 дцд╜дждндчдъ #T35 #_3═в┴ў_3╡ў╬е
 дцд╜дждодчджд╖ду #T35 #_3═в┴ў_5╢╚╝╘
 дцд╜джд▒дддлдп #T35 #_3═в┴ў_4╖╫▓ш
@@ -66351,7 +66350,7 @@
 дцд╙д╡д╖длдпд╦дє #T35 #_4╗╪║╣д╖_4│╬╟з
 дцд╙дтд╕ #T35 #_2╗╪_2╩╕╗·
 дцд╙дядтд╬дмд┐дъ #T35 #_3╗╪╬╪_5╩к╕ь
-дцд▐дпддд╕дчдж #T35 #_3╠¤╦ь_4░╩╛х
+дцд▐дпддд╕дчдж #T35 #_3╠¤╦ь_4░█╛я
 дцд▐дпд▒ддд╗дддъдчдп #T35 #_3╠¤╦ь_4╖┴└о_3╬╧
 дцдрдщдкдєд╗дє #T35 #_3┼Є┬╝_4▓╣└Ї
 дцдсдлдлдп #T35 #_2╠┤_3▓┴│╩
